platforms/intel-legacy/broadwell/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _platform-broadwell:
+
+Intel Broadwell
+###############
+
+Intel Broadwell platform is based on an core CPU and a Tensilica HiFi
+EP DSP integrated in the PCH. It relies on both internal memory (no
```

Comment:
Line 7: Not sure what is meant here: "... relies on both internal ... "

Reply:
Likely meant "only" instead of "on both".

Reply:
yes, copy/paste, it's 'only'